messaging/README.md

```diff
@@ -0,0 +1,11 @@
+# Messaging Interface
+This is the interface for the `wasmcloud:messaging` contract. This contract is a very simple abstraction over the concept of a message broker. This contract does not have controls or knobs or settings for things like delivery guarantees (e.g. "at most once" vs "at least once"), persistence of messages, etc. 
+
+Any details like that are the responsibility of individual providers.
```

Comment:
start the second paragraph with "This contract does not ...", and the "Any details ..." sentence is in the same paragraph above